x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -4,83 +4,113 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { memo, useState, useMemo } from 'react';
+import { memo, useState, useMemo, useCallback } from 'react';
 import React from 'react';
-import { EuiDataGrid } from '@elastic/eui';
-import { useSelector } from 'react-redux';
+import { EuiDataGrid, EuiDataGridColumn, EuiPage, EuiPageBody, EuiPageContent } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import * as selectors from '../../store/selectors';
+import { useHistory } from 'react-router-dom';
+import * as selectors from '../../store/alerts/selectors';
 import { usePageId } from '../use_page_id';
+import { useAlertListSelector } from '../../store/alerts/alert_hooks';
 
 export const AlertIndex = memo(() => {
   usePageId('alertsPage');
+  const history = useHistory();
 
-  const columns: Array<{ id: string }> = useMemo(() => {
+  const columns: EuiDataGridColumn[] = useMemo(() => {
     return [
-      { id: 'alert_type' },
-      { id: 'event_type' },
-      { id: 'os' },
-      { id: 'ip_address' },
-      { id: 'host_name' },
-      { id: 'timestamp' },
-      { id: 'archived' },
-      { id: 'malware_score' },
+      { id: 'Alert Type' },
```

Comment:
You can probably keep the `id`'s the same as they were before. To provide friendly names the column headers you can use the `EuiDataGridColumn` `display` property. So you could do something like:
```
{
    id: 'event_type',
    display: i18n.translate(
        'xpack.endpoint.application.endpoint.alerts.eventType',
        {
          defaultMessage: 'Event Type',
        }
}
```